### ISSUE TITLE
Left nav in mini-mode is missing alt text for icons

### ISSUE THREAD

**Issue by jwforres** [openshift/origin-web-console#2294]
When you click the hamburger to collapse the left nav, the text is set to display:none which makes it invisible to a screen reader.  There is no alternative text for the icons in this case.

**Comment by spadgett** [comment]
This should probably be addressed in patternfly. They make `.list-group-item-value` hidden, but maybe it should be `.sr-only()` instead.
